src/video/sdl2_v.cpp

```diff
@@ -520,6 +521,31 @@ bool VideoDriver_SDL_Base::PollEvent()
 				/* mouse left the window, undraw cursor */
 				UndrawMouseCursor();
 				_cursor.in_window = false;
+			} else if (ev.window.event == SDL_WINDOWEVENT_FOCUS_GAINED) {
+				this->window_has_focus = true;
```

Comment:
Reopen gamepad by its id after window has gained focus.
```suggestion
				this->window_has_focus = true;
				if (this->gamepad != nullptr) {
					SDL_JoystickID joyid = SDL_JoystickInstanceID(SDL_GameControllerGetJoystick(this->gamepad));
					this->CloseGamepad();
					this->OpenGamepadByJoystickID(joyid);
				}
```